src/scripts/main.js

```diff
@@ -355,6 +355,25 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people);
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const { names, sex, born, died } = person;
```

Comment:
It seems you have a typo in your code. You are trying to destructure the `names` property from the `person` object, but there is no such property in your object. It should be `name` instead of `names`.